lib/utils.js

```diff
@@ -30,4 +31,116 @@ function inspectObject (args) {
   }
 }
 
-export { inspectObject };
+/**
+ * Takes the caps that were provided in the request and translates them
+ * into caps that can be used by the inner drivers.
+ * @param {*} jsonwpCaps
+ * @param {*} w3cCapabilities
+ * @param {*} constraints
+ * @param {*} defaultCapabilities
+ */
+function parseCapsForInnerDriver (jsonwpCaps, w3cCapabilities, constraints={}, defaultCapabilities={}) {
+  // Check if the caller sent JSONWP caps, W3C caps, or both
+  const hasW3CCaps = _.isObject(w3cCapabilities);
+  const hasJSONWPCaps = _.isObject(jsonwpCaps);
+
```

Comment:
should we verify, that at least any of these caps is set

Reply:
We don't need to. The route handler validates the parameters so it's impossible for none of them to be set.